src/native/libs/Common/pal_utilities.h

```diff
@@ -12,7 +12,9 @@
 #include <stdio.h>
 #include <stdbool.h>
 #include <string.h>
-#include <unistd.h>
+#ifndef  _WIN32
+    #include <unistd.h>
+#endif
```

Comment:
This file is available on all recent Windows compiler versions, so excluding is no longer strictly necessary as it was a few years ago.

Reply:
I did try removing it but I would get many failures, unfortunately, so decided to keep it as attempting to fix it was out of the scope of the fix.